apps/namadillo/src/App/Settings/EnableFeatures.tsx

```diff
@@ -0,0 +1,47 @@
+import { Stack } from "@namada/components";
+import { FeaturesCheckbox } from "./FeaturesCheckbox";
+
+export const EnableFeatures = (): JSX.Element => {
+  return (
+    <Stack gap={2}>
+      <h3 className="mb-4 text-sm">Click to enable next phases features</h3>
+      <Stack as="ul" gap={3}>
+        <li>
+          <FeaturesCheckbox
```

Comment:
I'm wondering whether it might make more sense to have a single setting for the phase rather than individual features that can be toggled, but I'm not sure yet. I will talk to you about it :smile: